trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_naca_4series.py

```diff
@@ -18,147 +20,67 @@ def compute_naca_4series(camber,camber_loc,thickness,npoints=100,airfoilName='na
     None
 
     Inputs:
-    camber      [-] Maximum camber as a fraction of chord
-    camber_loc  [-] Maximum camber location as a fraction of chord
-    thickness   [-] Maximum thickness as a fraction of chord
-    npoints     [-] Total number of points
+        airfoils              <string>
 
     Outputs:
     airfoil_data.
-        thickness_to_chord 
-        x_coordinates 
-        y_coordinates
-        x_upper_surface
-        x_lower_surface
-        y_upper_surface
-        y_lower_surface
-        camber_coordinates  
+        thickness_to_chord    [unitless]
+        x_coordinates         [meters]
+        y_coordinates         [meters]
+        x_upper_surface       [meters]
+        x_lower_surface       [meters]
+        y_upper_surface       [meters]
+        y_lower_surface       [meters]
+        camber_coordinates    [meters] 
 
     Properties Used:
     N/A
-    """        
-    
-    airfoil_data                    = Data()
-    airfoil_data.x_coordinates      = Data()
-    airfoil_data.y_coordinates      = Data()
-    airfoil_data.thickness_to_chord = Data()
-    airfoil_data.camber_coordinates = Data()
-    airfoil_data.x_upper_surface    = Data()
-    airfoil_data.x_lower_surface    = Data()
-    airfoil_data.y_upper_surface    = Data()
-    airfoil_data.y_lower_surface    = Data()
-    
-    aNames    = [airfoilName]
-    half_pnts = int(npoints/2)
-    
-    upper = []
-    lower = []
+    """         
+    geometry        = Data()
+    half_npoints    = math.floor(npoints/2)      # number of points per side    
+    airfoil_digits  = [int(x) for x in airfoil_geometry_file]
+    camber          = airfoil_digits[0]/100 #   Maximum camber as a fraction of chord
+    camber_loc      = airfoil_digits[1]/10  #   Maximum camber location as a fraction of chord
+    thickness       = airfoil_digits[2]/10 + airfoil_digits[3]/100 #   Maximum thickness as a fraction of chord  
+ 
+    te = 1.5                                    # points per side and trailing-edge bunching factor
```

Comment:
default this in function definition

Reply:
I changed it from entering the digits separately so that all the airfoil import functions were consistent. see plot_vehicle and propeller_design.py as an example